command/file_test.go

```diff
@@ -0,0 +1,20 @@
+package command
+
+import (
+	"os/exec"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestCopyFileErrorIfDirectory(t *testing.T) {
+	t.Log("It fails if source is a directory")
+	{
+		cmd := exec.Command("mkdir", "source_dir")
```

Comment:
please create a tmp dir for testing (pathutil.NormalizedOSTmpDir...)
do not forget to check errors!